lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyProjectsManager.java

```diff
@@ -94,4 +94,7 @@ public void cleanUpTempDirs() {
         }
     }
 
+    public void cleanInstance() {
+        libertyWorkspaceFolders = new ArrayList<LibertyWorkspace>();
```

Comment:
Should this call `cleanUpTempDirs()` before reinitializing the array to empty?

Reply:
Calling `cleanInstance()` in `stop()` in `LibertyExtension` after `cleanUpTempDirs()`.